front-end/header.js

```diff
@@ -0,0 +1,3 @@
+// Copyright 2019-2020 @Premiurly/polkassembly authors & contributors
```

Comment:
Please update the header for the current repo.

Reply:
Where are we setting up the antd styling and custom theme for antd and tailwind ?